lib/fetch/index.js

```diff
@@ -122,130 +124,170 @@ class Fetch extends EE {
   }
 }
 
+const subscribed = (name) => {
+  if (channels[name].hasSubscribers) {
+    channels[name].publish(context)
+  }
+}
+const callAsyncStartEnd = () => {
+  subscribed('asyncStart')
+  subscribed('asyncEnd')
+}
```

Comment:
please use top level functions for those

Reply:
helper functions are now top level